src/edn/mod.rs

```diff
@@ -223,7 +223,7 @@ impl core::fmt::Display for Vector {
             "[{}]",
             self.0
                 .iter()
-                .map(|i| format!("{:?}, ", i))
+                .map(|i| format!("{}, ", i))
```

Comment:
I don't think I understood why those were removed?

Also, this file is using lots of `Edn::Key` without a `:` prefix on the `String`, we should add that to make the use consistent

Reply:
because this was including Edn::Type in the Display. So, for type `Edn::Key(:something)`: 

* `.map(|i| format!("{:?}, ", i))` becomes `Edn::Key(:something)`
* `.map(|i| format!("{}, ", i))` becomes `:something`

Reply:
if we remove the `:` `.map(|i| format!("{}, ", i))` becomes `something`, which would be wrong